### DISCUSSION TITLE
Local Web Browser Not Starting Properly with Latest Update

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#755] [Bug Reports]
I tried the latest image updated to 20220107 on an RPi4 with a 2.8 Capacitive touch screen. When I startup, the local screen says, "Page Unresponsive." When I click on "Exit Page", I get a screen that says "Aw, Snap! Something went wrong while displaying this webpage. Error code: SIGTRAP." After I click on "Reload", rAudio loads normally. 

I add the following code to config.txt for this screen:
`dtparam=spi=on
dtparam=i2c1=on
dtparam=i2c_arm=on
dtoverlay=pitft28-capacitive,rotate=270,touch-swapxy,touch-invy,speed=32000000,fps=20`

Clicking on the gear icon for Chromium in Browser on RPi under Features I see the following: 
`Jan 13 15:18:14 rAudio sudo[1218]: [1218:1221:0113/151814.345486:ERROR:child_thread_impl.cc(345)] Crashing because hung`

I have been using this screen with no problems for years--this is the first time I have experienced this.

Any ideas what could be going on?

**Comment by rern** [comment]
Please download [the latest image](https://github.com/rern/rAudio-1/releases/tag/i20220116) file and install **Chromium downgrade** from **Addons**.

**Reply by feebb** [reply]
Thanks @rern. Tried the new image and downgraded Chromium. While it cured the error for Chromium in Browser on RPi under Features, I still get the Page Unresponsive and Aw, Snap pages on startup. After I reload from the Aw Snap page everything works. Is there anything else I can try?
